src/api/mapping.ts

```diff
@@ -22,4 +22,29 @@ mappingRouter.get('/', (req, res) => {
   res.json(domains)
 })
 
+mappingRouter.delete('/delete/:id', (req, res) => {
+  const domains = getMappings()
+  const deletedDomain = domains.find((e) => e.id === req.params.id)
+  const updatedDomains = domains.filter((e)=>{
+    return e.id !== req.params.id
+  })
+  setData('mappings', updatedDomains)
+  res.json(deletedDomain)
+})
+
+mappingRouter.patch('/edit/:id', (req, res) =>{
+  const domains = getMappings()
+  const editDomain = domains.map((element: Mapping) => {
```

Comment:
`domainList`

Reply:
`editDomain` implies 1 domain. This is an array, so `domainList` is a better name.

Reply:
or `newDomainList`